src/FreeBirdIO.jl

```diff
@@ -196,20 +200,23 @@ function convert_walker_to_system(at::AtomWalker)
     config::FastSystem = at.configuration
     energy = at.energy.val
     iter = at.iter
-    num = at.num_frozen_part
+    num = join(at.list_num_par, ",")
+    frozen = join(at.frozen, ",")
     e_frozen = at.energy_frozen_part.val
-    return AbstractSystem(config; energy=energy, iter=iter, num_frozen_part=num, energy_frozen_part=e_frozen)
+    return AbstractSystem(config; energy=energy, iter=iter, list_num_par=num, frozen=frozen, energy_frozen_part=e_frozen)
 end
 
 """
     append_walker(filename::String, at::AtomWalker)
 
 Append an `AtomWalker` object to a file.
 """
-function append_walker(filename::String, at::AtomWalker)
-    ats = read_walkers(filename)
-    push!(ats, at)
-    write_walkers(filename, ats)
+function append_walker(filename::String, at::AtomWalker{C}) where C
+    # ats = read_walkers(filename)
+    sys = convert_walker_to_system(at)
+    # push!(ats, at)
```

Comment:
@yangmr04, do you want to keep these comments around?

Reply:
Gone now!